rules/integrations/kubernetes/execution_pod_created_in_kube_namespace.toml

```diff
@@ -0,0 +1,56 @@
+[metadata]
+creation_date = "2022/05/23"
+maturity = "production"
+updated_date = "2022/05/23"
+integration = "kubernetes"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects when a pod is created in the kube-system or kube-public namespace. These default namespaces are reserved for cluster services and cannot be deleted. Only system administrators should edit resources here and as a best practice user resources should not be deployed in either 'kube' namespace. 
+
+The kube-system namespace is reserved for objects created by the Kubernetes system. Adversaries can use this namespace to deploy malicious pods or containers with names similar to those of default system components in order to evade detection. 
+
+The kube-public namespace is intended for public resources and unauthenticated user access. A user pod created here should be further investigated to determine its validity in the cluster. 
+
+Adversaries can deploy malicious pods in these namespaces to maintain persistence, evade detection, or facilitate execution in a cluster. 
+"""
+false_positives = [
+    """
+    There are legitimate uses of these namespaces that could trigger alerts. The Kubernetes systems deploys objects to kube-system namespace and administrators may deploy objects to kube-public that are intended for public access.
+    """,
+]
+index = ["filebeat-*", "logs-kubernetes.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Pod Created in Kube Namespace"
+note = """## Config
+
+The Kubernetes Fleet integration with Audit Logs enabled, Filebeat module, or similarly structured data is required to be compatible with this rule."""
+references = ["https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/",
+"https://media.defense.gov/2021/Aug/03/2002820425/-1/-1/0/CTR_Kubernetes_Hardening_Guidance_1.1_20220315.PDF"]
+risk_score = 47
+rule_id = "d62d3782-40f5-4d25-99d5-ff95f094c043"
+severity = "medium"
+tags = ["Elastic", "Kubernetes", "Continuous Monitoring", "Execution"]
+timestamp_override = "event.ingested"
```

Comment:
are the lookback and intervals good with the default values for this? Or should this be slowed down

```
defaults:
lookback (to - from) = 6m (now - now-6m)
interval = 5m
```

Reply:
I'm not sure, what was your reasoning?

Reply:
@brokensound77

Reply:
Any logging coming from a cloud service (or RESTful API) tends to come in batches and we have observed that some of those defaults can be slower (such as every 10-30mins). 

The rule defaults would be to search `6m` back every `5m` , but if the logs come every 30 mins, then that is 6 rule executions over a period where only 1 execution would be expected to find new items (as an example).

With `event.ingested`, it is an approach to be resilient to such delays, by looking at the time the event came in, rather than occurred. So in some cases we can improve performance by slowing down the rule executions (increasing window and interval).

Since this is on an index that could potentially be local or cloud based, we may be ok with the defaults just to be more expedient. 

Hope this helps - this is a good thing to consider for all rules

Reply:
Yes, very helpful explanation Thank you!